packages/eslint-plugin/src/rules/explicit-member-accessibility.ts

```diff
@@ -15,30 +37,132 @@ export default util.createRule({
     messages: {
       missingAccessibility:
         'Missing accessibility modifier on {{type}} {{name}}.',
+      unwantedPublicAccessibility:
+        'Public accessibility modifier on {{type}} {{name}}.',
     },
-    schema: [],
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          accessibility: accessibilityLevel,
+          overrides: {
```

Comment:
add `additionalProperties: false` to the overrides object

Reply:
done